training/model_management/components/import_model/asset.yaml

```diff
@@ -1,2 +1,3 @@
 type: component
 spec: spec.yaml
+categories: ["Import Model"]
```

Comment:
Are we listing broad level categories somewhere?

Reply:
Hi @elesel could you please help to respond to above query?

Reply:
Thanks @elesel. Updated categories to `Models`